packages/nimble-components/src/patterns/button/styles.ts

```diff
@@ -254,87 +262,163 @@ export const styles = css`
 
 export const buttonAppearanceVariantStyles = css``.withBehaviors(
     appearanceBehavior(
-        ButtonAppearance.outline,
+        ButtonAppearance.block,
```

Comment:
Is there a reason you swapped the order of the outline and block appearanceBehaviors? It makes the diff harder to make sense of.